pkg/koordlet/metricsadvisor/collector.go

```diff
@@ -232,8 +236,8 @@ func (c *collector) collectPodResUsed() {
 		}
 		klog.V(6).Infof("collect pod %s/%s, uid %s finished, metric %+v",
 			meta.Pod.Namespace, meta.Pod.Name, meta.Pod.UID, podMetric)
-		err := c.metricCache.InsertPodResourceMetric(collectTime, &podMetric)
-		if err != nil {
+
+		if err := c.metricCache.InsertPodResourceMetric(collectTime, &podMetric); err != nil {
 			klog.Infof("insert pod %s/%s, uid %s resource metric failed, metric %v, err %v",
```

Comment:
here should replace klog.Infof with klog.Errorf

Reply:
Fixed.